src/fontra/client/core/fontra-menus.js

```diff
@@ -0,0 +1,222 @@
+    } else if (mapMenuItemKeyToFunction[itemKey]) {
+      menuBarArray.push(mapMenuItemKeyToFunction[itemKey](delegate));
+    } else {
+      console.log("Method/Function does not exist, skip: ", itemKey, methodName);
+    }
+  }
+
+  menuBarArray.push(getHelpMenuItems()); // Help-Menu at the end.
+  const menuBar = new MenuBar(menuBarArray);
+  return menuBar;
+}
+
+function getFontraMenuItems() {
+  return {
+    title: "Fontra",
+    bold: true,
+    getItems: () => {
+      const menuItems = [
+        "shortcuts",
+        "theme-settings",
+        "display-language",
+        "clipboard",
+        "editor-behavior",
+        "plugins-manager",
+        "server-info",
+      ];
+      return menuItems.map((panelID) => ({
+        title: translate(`application-settings.${panelID}.title`),
+        enabled: () => true,
+        callback: () => {
+          window.open(`/applicationsettings/applicationsettings.html#${panelID}-panel`);
+        },
+      }));
+    },
+  };
+}
+
+function getHelpMenuItems() {
+  return {
+    title: translate("menubar.help"),
+    getItems: () => {
+      return [
+        {
+          title: translate("menubar.help.homepage"),
+          enabled: () => true,
+          callback: () => {
+            window.open("https://fontra.xyz/");
+          },
+        },
+        {
+          title: translate("menubar.help.documentation"),
+          enabled: () => true,
+          callback: () => {
+            window.open("https://docs.fontra.xyz");
+          },
+        },
+        {
+          title: translate("menubar.help.changelog"),
+          enabled: () => true,
+          callback: () => {
+            window.open("https://fontra.xyz/changelog.html");
+          },
+        },
+        {
+          title: "GitHub",
+          enabled: () => true,
+          callback: () => {
+            window.open("https://github.com/googlefonts/fontra");
+          },
+        },
+      ];
+    },
+  };
+}
+
+function getFileMenuItems(delegate) {
+  return {
+    title: translate("menubar.file"),
+    getItems: () => {
+      let exportFormats =
+        delegate.fontController?.backendInfo.projectManagerFeatures["export-as"] || [];
+      if (exportFormats.length > 0) {
+        return [
+          {
+            title: translate("menubar.file.export-as"),
+            getItems: () =>
+              exportFormats.map((format) => ({
+                actionIdentifier: `action.export-as.${format}`,
+              })),
+          },
+        ];
+      } else {
+        return [
+          {
+            title: translate("menubar.file.new"),
+            enabled: () => false,
+            callback: () => {},
+          },
+          {
+            title: translate("menubar.file.open"),
+            enabled: () => false,
+            callback: () => {},
+          },
+        ];
+      }
+    },
+  };
+}
+
+// TODO: Is a default possible, or do we skip it if not provided by the 'View'?
+// function getEditMenuItems() {
```

Comment:
Yes, it should be populated with action identifiers.

Reply:
... and fallback default titles, as commented in the other comment thread.

Reply:
I added action identifiers for `getEditMenuItems` and `getViewMenuItems` as defaults. I am closing this.